$MirOS: ports/multimedia/mplayer/patches/patch-libmpdemux_tvi_bsdbt848_c,v 1.1.7.1 2005/03/18 15:50:49 tg Exp $
--- libmpdemux/tvi_bsdbt848.c.orig	Mon Dec 27 17:29:52 2004
+++ libmpdemux/tvi_bsdbt848.c	Thu May 19 20:32:10 2005
@@ -40,7 +40,7 @@
 #include <string.h>
 
 #include <sys/param.h>
-#ifdef __NetBSD__
+#if defined(__NetBSD__) || defined(__OpenBSD__)
 #include <dev/ic/bt8xx.h>
 #include <sys/audioio.h>
 #elif __FreeBSD_version >= 502100
@@ -572,7 +572,7 @@ if(priv->tunerfd < 0)
 /* Audio Configuration */
 
 priv->dspready = TRUE;
-#ifdef __NetBSD__
+#if defined(__NetBSD__) || defined(__OpenBSD__)
 priv->dspdev = strdup("/dev/sound");
 #else
 priv->dspdev = strdup("/dev/dsp");
